graylog2-web-interface/src/pages/ConfigurationsPage.tsx

```diff
@@ -178,6 +179,14 @@ const ConfigurationsPage = () => {
       showCaret: true,
       catchAll: true,
     },
+    {
+      name: 'Keyboard Shortcuts',
+      SectionComponent: ConfigurationSection,
+      props: {
+        ConfigurationComponent: KeyboardShortcutsSection,
+        title: 'Graylog Keyboard Shortcuts',
+      },
+    },
```

Comment:
I am not sure if the configurations page is the best place for this page, since we currently do not plan to make the shortcuts configurable. We could also create a separate page, with a separate route and add a link in the dropdown for the help icon.

![image](https://github.com/Graylog2/graylog2-server/assets/46300478/0fa0ea15-4345-45e2-b967-36679fc7b2e2)